src/main/java/org/fundacionjala/coding/mary/InnerContent.java

```diff
@@ -0,0 +1,49 @@
+package org.fundacionjala.coding.mary;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+/**
+ * Created by Mary Ricalde on 10/1/2019.
+ */
+public final class InnerContent {
+    private static final int WORD_LENGTH = 3;
+
+    private InnerContent() {
+    }
+
+    public static String sortTheInnerContent(final String words) {
+        String result;
+
+        if (words == null) {
+            result = "Entered null string";
+            return result;
+        }
+
+        if (words.length() > 0) {
```

Comment:
This validation is redundant.
the for loop already performs that validation.

Reply:
Thanks for the comments, I did the updates.